### PR TITLE
 Add views

### PR DESCRIPTION
**Pull request by farouqdaif1**
## What does this PR do?
This PR is raised to Use migration files to build  schema.

## Description of the task to be completed
* [x]  Implement the design from the sneak peek wireframes
* [x]  A method that updates the likes counter for a post.
* [x]  Use methods that you have created in the [Blog App - processing data in models](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Processing_data_model_project.md).
## How should it be tested
Clone the repository

```
git clone https://github.com/farouqdaif1/blogapp.git
```

Navigate to the folder 
```
cd blogapp
```
Checkout the branch

```
git checkout views
```

### CODE REVIEW

**Review by uwadonat — CHANGES_REQUESTED** [review]
Hi **[farouqdaif1](https://github.com/farouqdaif1),** 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that does not have Linter . 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._
```

File: README.md [hunk 1]
```diff
@@ -43,3 +43,17 @@ Give a ⭐️ if you like this project!
...
+first_post = Post.create(author: first_user, title: 'Hello1', text: 'This is first post')
+second_post = Post.create(author: first_user, title: 'Hello2', text: 'This is second post')
+third_post = Post.create(author: second_user, title: 'Hello3', text: 'This is third post')
+fourth_post = Post.create(author: second_user, title: 'Hello4', text: 'This is fourth post')
+Comment.create(post: first_post, author: second_user, text: 'Hi Tom1!' )
+Comment.create(post: first_post, author: second_user, text: 'Hi Tom2!' )
+Comment.create(post: first_post, author: second_user, text: 'Hi Tom3!' )
+Comment.create(post: first_post, author: second_user, text: 'Hi Tom4!' )
+Comment.create(post: first_post, author: second_user, text: 'Hi Tom5!' )
+Comment.create(post: first_post, author: second_user, text: 'Hi Tom6!' )
```
Comment (GzimAsani) [inline README.md]: - [ ] I think you added this by mistake, kindly remove it

File: README.md [hunk 2]
```diff
@@ -43,3 +43,17 @@ Give a ⭐️ if you like this project!
 - Hat tip to anyone whose code was used
```
Comment (GzimAsani) [inline README.md]: - [ ] Please explain in more detail how a user can clone and start your project, include how it should create and migrate the databases

**Review by GzimAsani — CHANGES_REQUESTED** [review]
### Hi @farouqdaif1 and @uwadonat 
 
 Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there!
 
 ## Highlights
 * Your code organization is good.
 
 ## Required Changes 
 - [ ] Code implementation is good, but when I start the app nothing renders on my first page, please fix this
 - [ ] _Check inline comments below :arrow_double_down: _




 _Check the comments under the review._
 
 ## Optional suggestions
 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
 
Cheers and Happy coding! clapclapclap
 
 Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
 
 **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by t-yanick — APPROVED** [review]
Hi @farouqdaif1, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:  
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**